views/sql.form.view.php

```diff
@@ -9,6 +9,9 @@
 $url = (new Curl())
     ->setArgument('action', 'sqlexplorer.form')
     ->getUrl();
+
+define('ZBX_DB_ORACLE', 3); // Dummy value. Oracle DB support was dropped. With this change it works on Zabbix 7.2.3
```

Comment:
For backward compatibility i would suggest to change this line to something like:
`defined('ZBX_DB_ORACLE') or define('ZBX_DB_ORACLE', '');`